src/components/LatestGuests/GuestItems.js

```diff
@@ -0,0 +1,55 @@
+import React, { useState } from "react";
+import { Col, Modal, Button } from "react-bootstrap";
+import "./LatestGuests.css";
+
+const GuestItems = ({ img, buttonText, petName, petAge, petBreed, id }) => {
+  const [show, setShow] = useState(false);
+
+  const handleClose = () => setShow(false);
```

Comment:
this function and the following one can be done in 1 function which take param in order to change the setshow

Reply:
created 1 function which handles the setShow and it is working on the modal for show and close